Procfile

```diff
@@ -0,0 +1 @@
+worker: bundle exec sidekiq -C ./config/sidekiq.yml
```

Comment:
Minor: It's common practice to ensure there is a newline at the end of source files, e.g. [Style/FinalNewline](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/FinalNewline) and GitHub has highlighted the missing newline with a red symbol (see above).